src/main/java/slimeknights/tconstruct/tools/TinkerMaterials.java

```diff
@@ -15,39 +15,60 @@
-  public static final ItemObject<Block> manyullynBlock = BLOCKS.register("manyullyn_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
-  public static final ItemObject<Block> knightSlimeBlock = BLOCKS.register("knightslime_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
-  public static final ItemObject<Block> pigironBlock = BLOCKS.register("pigiron_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
+  public static final ItemObject<Item> copperNugget = ITEMS.register("copper_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> copperIngot = ITEMS.register("copper_ingot", GENERAL_PROPS);
   public static final ItemObject<Block> copperBlock = BLOCKS.register("copper_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
+
+  public static final ItemObject<Item> bronzeNugget = ITEMS.register("bronze_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> bronzeIngot = ITEMS.register("bronze_ingot", GENERAL_PROPS);
+  public static final ItemObject<Block> bronzeBlock = BLOCKS.register("bronze_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
+
+  public static final ItemObject<Item> ravagerSteelNugget = ITEMS.register("ravager_steel_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> ravagerSteelIngot = ITEMS.register("ravager_steel_ingot", GENERAL_PROPS);
+  public static final ItemObject<Block> ravagerSteelBlock = BLOCKS.register("ravager_steel_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
+
+  public static final ItemObject<Item> pigironNugget = ITEMS.register("pigiron_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> pigironIngot = ITEMS.register("pigiron_ingot", GENERAL_PROPS);
+  public static final ItemObject<Block> pigironBlock = BLOCKS.register("pigiron_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
+
+  public static final ItemObject<Item> roseGoldNugget = ITEMS.register("rose_gold_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> roseGoldIngot = ITEMS.register("rose_gold_ingot", GENERAL_PROPS);
   public static final ItemObject<Block> roseGoldBlock = BLOCKS.register("rose_gold_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
 
-  /*
-   * Items
-   */
-  // metals
+  public static final ItemObject<Item> soulSteelNugget = ITEMS.register("soul_steel_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> soulSteelIngot = ITEMS.register("soul_steel_ingot", GENERAL_PROPS);
+  public static final ItemObject<Block> soulSteelBlock = BLOCKS.register("soul_steel_block", GENERIC_METAL_BLOCK, GENERAL_TOOLTIP_BLOCK_ITEM);
+
+  public static final ItemObject<Item> alexandritePearl = ITEMS.register("alexandrite_pearl", GENERAL_PROPS);
+
+  public static final ItemObject<Item> heptazionNugget = ITEMS.register("heptazion_nugget", GENERAL_PROPS);
+  public static final ItemObject<Item> heptazionIngot = ITEMS.register("heptazion_ingot", GENERAL_PROPS);
```

Comment:
I can send you better temp textures for these if you wish. Its trivial to hue shift an iron ingot using the colors you assigned to the material/fluid, better for now than solid textures